### PR TITLE
Completed the task

### PR DESCRIPTION
**Pull request by Sonder1966**
Is this correct?

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
+teeth.fallOut();
  
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: doctor
+//  action 1: doctors cure people
+doctors.curepeople();
```
Comment (matkatmusic) [inline main.cpp]: the action pseudo-code is missing `camelCase` spelling.
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 1:
-
-//  action 2:
+//  Noun: fingers
+//  action 1: fingers bend at the joints
+fingers.joints();
+//  action 2: fingers grip things
+fingers.grip();
+//  action 3:people count on fingers
+fingers.count();
```
Comment (matkatmusic) [inline main.cpp]: `people` is the noun in your plain-english action, not `fingers`.
You need to revise your plain-english action
- [x] revise

File: main.cpp [hunk 3]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
+computers.binaryCode();
+
+//  6) 
+//  Noun: eye
+//  action 1: the eye uses light to see
+eye.useslight();
+//  action 2: eyes can open and close
+eye.openClose();
+//  action 3: some eyes need glasses to see
+eye.seeGlasses();
```
Comment (matkatmusic) [inline main.cpp]: your pseudo code doesn't match your english.
the verb in your plain-english expression is `need`, not `see`
- [x] revise

File: main.cpp [hunk 4]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 3:
- 
-//  6)
-//  Noun:
-//  action 1:
+//  Noun:computer
+//  action 1:computers can process numbers
+computers.processNumbers();
+//  action 2:computers can store data
+computers.storeData();
```
Comment (matkatmusic) [inline main.cpp]: your plain english statements express an ability to do something, not actually doing something.
`computers process numbers` and `computers store data` is what you should have written for the plain-english expression
- [x] revise

File: main.cpp [hunk 5]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: teeth
+//  action 1: teeth are used for chewing
+teeth.chewing();
```
Comment (matkatmusic) [inline main.cpp]: This plain-english expression is difficult to express as pseudo-code.
a better choice would have been `teeth chew food`
- [x] revise

File: main.cpp [hunk 6]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
+//  action 2: teeth get cavaties
+teeth.cavaties();
+//  action 3: first teeth fall out
+teeth.fallOut();
```
Comment (matkatmusic) [inline main.cpp]: your plain-english noun `first teeth` does not match your original noun: `Teeth`
- [x] revise

File: main.cpp [hunk 7]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: teeth
+//  action 1: teeth are used for chewing
+teeth.chewing();
+//  action 2: teeth get cavaties
+teeth.cavaties();
```
Comment (matkatmusic) [inline main.cpp]: the action here is `get`.   `cavities` is not a verb. 
- [x] revise

File: main.cpp [hunk 8]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 3:
- 
 //  4)
-//  Noun:
-//  action 1:
+//  Noun: fish
+//  action 1: fish swim in water
+fish.swim();
+//  action 2: fish can be eaten 
+fish.eaten();
```
Comment (matkatmusic) [inline main.cpp]: the verb here is `can be`, which expresses an ability.   these are difficult to express as code.
pick a different action that doesn't use `can` 
- [x] revise

File: main.cpp [hunk 9]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  6)
-//  Noun:
-//  action 1:
+//  Noun:computer
+//  action 1:computers can process numbers
+computers.processNumbers();
+//  action 2:computers can store data
+computers.storeData();
+//  action 3: computers use binary code 
+computers.binaryCode();
```
Comment (matkatmusic) [inline main.cpp]: `use` is the action here.  it is missing from your plain-english
- [ ] revise

File: main.cpp [hunk 10]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 3:
- 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: fingers
+//  action 1: fingers bend at the joints
+fingers.joints();
```
Comment (matkatmusic) [inline main.cpp]: the verb here is `bend`.  `joints` is a noun, not a verb.
your pseudo-code has no action.
- [x] revise

File: main.cpp [hunk 11]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 3:
- 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun:cloths
+//  action 1: everybody wears cloths
+cloths.wearsEverybody();
```
Comment (matkatmusic) [inline main.cpp]: `everybody` is the noun in your `action` statement.
your pseudo code says that cloths wear everybody. 
You need to change your action to be something that clothing does, such as 'flap in the wind'
- [x] revise

File: main.cpp [hunk 12]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
+cloths.fashion();
 
-//  action 3:
- 
 //  9)
-//  Noun:
-//  action 1:
+//  Noun: televison
+//  action 1: everybody watches televison
+televison.watches();
```
Comment (matkatmusic) [inline main.cpp]: the noun in your plain-english expression is `everybody`
change your plain english action to something that the television does.
- [x] revise

File: main.cpp [hunk 13]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
+//  action 2: televisions make sound 
+televison.sound();
+//  action 3: televisions broadcast programs
+televison.broadcast();
```
Comment (matkatmusic) [inline main.cpp]: televisions don't broadcast signals.  they display them.
revise your plain-english action and pseudo-code
- [x] revise

File: main.cpp [hunk 14]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-
-//  action 3:
- 
+//  Noun: clocks
+//  action 1:a clock displays the time
+clock.displayTime();
+//  action 2: a clock uses numbers
+clock.usesNumbers();
+//  action 3: some clocks hang on walls
+clock.hang();
```
Comment (matkatmusic) [inline main.cpp]: the full expression is `clock.hangOnWall();`
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

**Comment by Sonder1966** [comment]
I revised all corrections.

File: main.cpp [hunk 15]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 2:
-
-//  action 3:
+//  Noun: teeth
+//  action 1: teeth chew food
+teeth.chewFood();
+//  action 2: teeth get cavaties
+teeth.getCavaties();
+//  action 3: teeth are cleaned with a brush
+teeth.cleanedBrush();
```
Comment (matkatmusic) [inline main.cpp]: As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [x] come up with a new noun and 3 actions
- [x] pick a noun that needs `camelCase` for the name
- [x] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 15 above)
Comment (Sonder1966) [reply main.cpp]: OK thanks, I will address these corrections ASAP

Comment (Sonder1966) [reply main.cpp]: I have replaced this noun with guitarAmp

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**